frontend/src/app/attempts/[id]/page.jsx

```diff
@@ -0,0 +1,38 @@
+"use client"
+import React, { useEffect, useState } from 'react';
+import { useSession } from 'next-auth/react';
+import { getAttempt } from '@app/api/attemptsService';
+import AttemptReader from '@components/AttemptReader';
+
+const AttemptPage = ({ params }) => {
+    const [attemptData, setAttemptData] = useState(null);
+    const { data: session } = useSession();
+    const attemptId = params.id;
+
+    useEffect(() => {
+        if (session && attemptId) {
+            getAttempt(session.user.email, attemptId)
+                .then(response => {
+                    setAttemptData(response);
+                })
+                .catch(error => {
+                    console.error('Error fetching attempt:', error);
+                });
+        }
+    }, [session, attemptId]);
+
+    if (!attemptData?.data) return <div>No attempt data found.</div>;
+
+    return (
+        <>
+            <h1>This is attempt {attemptId}.</h1>
```

Comment:
I think maybe no need include attempt id bah, cos the number might be abit misleading to the user